packages/spindle-ui/src/HeroCarousel/HeroCarousel.css

```
@@ -0,0 +1,110 @@
+/* stylelint-disable plugin/selector-bem-pattern */
+.spui-HeroCarousel__CarouselContainer {
+  align-items: center;
+  display: flex;
+  height: 192px;
+  justify-content: center;
+  margin: -1px 0;
+  overflow: hidden;
+  padding: 1px 0;
+}
+
+.spui-HeroCarousel__List {
+  display: flex;
+  margin-right: 14px;
+  padding: 0 7px;
+  transition: transform 0.5s;
+  width: 280px;
+}
+
+.spui-HeroCarousel__Controls {
+  align-items: center;
+  border: 1px solid var(--color-border-low-emphasis);
+  border-radius: 100px;
+  display: flex;
+  height: 40px;
+  justify-content: center;
+  margin: 14px auto 0;
+  width: 122px;
+}
+
+.spui-HeroCarousel__Control {
+  background-color: transparent;
+  border: none;
+  height: 40px;
+  width: 40px;
+}
+
+.spui-HeroCarousel__Control:hover {
+  opacity: 0.7;
+}
+
+.spui-HeroCarousel__Control:nth-of-type(1) {
+  padding: 11px 10px 11px 12px;
+}
+
+.spui-HeroCarousel__Control:nth-of-type(2) {
+  border-left: 1px solid var(--color-border-low-emphasis);
+  border-right: 1px solid var(--color-border-low-emphasis);
+  padding: 11px;
+}
+
+.spui-HeroCarousel__Control:nth-of-type(3) {
+  padding: 11px 12px 11px 10px;
+}
+
+.spui-HeroCarousel__Control > svg {
+  color: var(--color-text-low-emphasis);
+  height: 18px;
+  width: 18px;
+}
+
+@media (prefers-reduced-motion) {
```

```suggestion
@media (prefers-reduced-motion: reduce) {
```
明示的にする方に合わせてたのを思い出しました
https://github.com/openameba/spindle/pull/242#discussion_r668585195

---

`reduce`付けましました！ 💯